examples/llama2/llama2_cpu.json

```diff
@@ -0,0 +1,110 @@
+{
+    "input_model":{
```

Comment:
Instead of separate json files for cpu and gpu, could we just use one template json file and generate needed json file in llama2.py based on inputs ?

Reply:
Updated.